src/dlgProfilePreferences.cpp

```diff
@@ -1632,8 +1632,7 @@ void dlgProfilePreferences::setCommandBgColor()
 void dlgProfilePreferences::setFontSize()
 {
     mFontSize = fontSize->currentIndex() + 1;
-    // delay setting pHost->mDisplayFont until save is clicked by the user.
-    //setDisplayFont();
+    setDisplayFont();
```

Comment:
Great improvement, but not strictly related to the aim of the PR - could you do it separately? 

But in a similar vein, the command line auto-adjusting as you edit the command line minimum height would be nice - right now it doesn't adjust on changing the value nor save, but on the first letter typed.

Reply:
> Great improvement, but not strictly related to the aim of the PR - could you do it separately?

Yes, I don't mind. I opted to include it since it made the testing of a variety of fonts and sizes much easier.

Reply:
Done, I cherry picked the commit and made this PR for the change: https://github.com/Mudlet/Mudlet/pull/5351

I reverted the commit on this branch, so there shouldn't be any conflicts between the two since this one no longer involves `dlgProfilePreferences.cpp`